src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerTest.php

```diff
@@ -209,6 +212,156 @@ public function testJsonWithSerializerContextOverride()
         $this->assertEquals('{}', $response->getContent());
     }
 
+    public function testFile()
+    {
+        /* @var ContainerInterface $container */
+        $container = $this->getMock('Symfony\Component\DependencyInjection\ContainerInterface');
+        $kernel = $this->getMock('Symfony\Component\HttpKernel\HttpKernelInterface');
+        $container->set('kernel', $kernel);
+
+        $controller = new TestController();
+        $controller->setContainer($container);
+
+        /* @var BinaryFileResponse $response */
+        $response = $controller->file(new File(__FILE__));
+
+        $this->assertInstanceOf(BinaryFileResponse::class, $response);
+        $this->assertEquals(200, $response->getStatusCode());
```

Comment:
You should use `assertSame` instead of `assertEquals` when possible to do a strict comparison.